lib/Interpreter/IncrementalParser.cpp

```diff
@@ -377,6 +377,18 @@ namespace cling {
     return ParseResultTransaction(T, ParseResult);
   }
 
+  std::string IncrementalParser::makeModuleName() {
+    stdstrstream ModuleName;
+    ModuleName << "cling-module-" << m_ModuleNo++;
+    return std::move(ModuleName.str());
```

Comment:
Yes better.
If they didn't make std::to_string so useless, I wouldn't always forget about it.
Added another commit to remove sstream & iostream includes.